Code/Engine/RendererFoundation/Descriptors/Enumerations.h

```diff
@@ -162,24 +166,42 @@ struct ezGALVertexAttributeSemantic
   };
 };
 
-/// \brief General type of buffer.
+/// \brief Defines for what purpose a buffer can be used for.
 /// \sa ezGALBufferCreationDescription
-struct ezGALBufferType
+struct ezGALBufferFlags
```

Comment:
Maybe we could call it BufferUsageFlags. I don't have a strong preference though.

Reply:
That works too, fixed!